src/react-components/room-info-dialog.js

```diff
@@ -1,118 +1,95 @@
-              </a>
-            </div>
-            <div className={styles.attributionAuthor}>
-              by {a.author} {source}
-            </div>
-          </div>
-        );
-      } else {
-        return (
-          <div className={styles.attribution} key={`${a.name} ${a.author} {i}`}>
-            <div className={styles.attributionName}>{a.name}</div>
-            <div className={styles.attributionAuthor}>by {a.author}</div>
-          </div>
-        );
-      }
-    };
+function SceneAttribution({ attribution }) {
+  if (attribution.url) {
+    const source = attribution.url.includes("sketchfab.com")
+      ? "on Sketchfab"
+      : attribution.url.includes("poly.google.com")
+        ? "on Google Poly"
+        : "";
 
-    let attributions = null;
-    let creator = null;
-
-    if (hasScene && this.props.scene.attributions) {
-      creator = this.props.scene.attributions.creator;
-      attributions = (
-        <div>
-          {this.props.scene.attributions.content && this.props.scene.attributions.content.map(toAttributionDiv)}
+    return (
+      <li className={styles.attribution}>
+        <div className={styles.attributionName}>
+          <a href={attribution.url} target="_blank" rel="noopener noreferrer">
+            {attribution.name}
+          </a>
         </div>
-      );
-    }
-
-    const title = (
-      <div className={styles.title}>
-        {hasDescription && hasScene ? <FormattedMessage id="room-info.title" /> : this.props.hubName}
-      </div>
+        <div className={styles.attributionAuthor}>
+          by {attribution.author} {source}
+        </div>
+      </li>
     );
-
+  } else {
     return (
-      <DialogContainer title={title} wide={true} {...this.props}>
-        {hasDescription && (
-          <>
-            {hasScene && <div className={styles.hubTitle}>{this.props.hubName}</div>}
-            <div className={styles.description}>{this.props.hubDescription}</div>
-          </>
-        )}
+      <li className={styles.attribution}>
+        <div className={styles.attributionName}>{attribution.name}</div>
+        <div className={styles.attributionAuthor}>by {attribution.author}</div>
+      </li>
+    );
+  }
+}
+
+// TODO: Move to /room folder
+export function RoomInfoDialog({ scene, store, hubName, hubDescription, onClose }) {
+  const hasScene = !!scene;
+  const showSceneLink = hasScene && allowDisplayOfSceneLink(scene, store);
+  const attributions = (scene && scene.attributions && scene.attributions.content) || [];
+  const creator = scene && scene.attributions && scene.attributions.creator;
+
+  return (
+    <Sidebar title={<FormattedMessage id="room-info.title" />} beforeTitle={<CloseButton onClick={onClose} />}>
+      <div className={styles.roomInfoDialog}>
+        <h2 className={styles.sectionTitle}>Room</h2>
+        {hubName && <InputField label="Name">{hubName}</InputField>}
+        {hubDescription && <InputField label="Description">{hubDescription}</InputField>}
```

Comment:
The text in this panel seems to be inheriting a value for `user-select` that isn't `text` -- or something else is preventing text highlighting/selection

Reply:
I found the culprit in the old `.ui` class and removed it. This makes a lot more things than I would like selectable though. I think we need to introduce styles/components for non-selectable labels, headings, images, and links. I think it's probably safer to opt-out of text selection than opt-in though. I'll follow up this PR with one to address this.